Packs/Emailrep/Integrations/EmailRep/EmailRep.py

```diff
@@ -0,0 +1,249 @@
+            request_params["expires"] = expires
+
+        demisto.debug(f"DDDD:  {request_params}")
+
+        return self._http_request(
+            method='POST',
+            url_suffix='/report',
+            json_data=request_params
+        )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Test API connectivity and authentication."""
+    try:
+        client.get_email_address_reputation(email="test@example.com")
+    except DemistoException as e:
+        if 'invalid api key' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return 'ok'
+
+
+def email_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Get email address reputation from EmailRep and calculate DBotScore.
+
+    DBot score:
+    Good: Suspicious = false
+    Malicious: Suspicious = true + (malicious_activity_recent = true or credentials_leaked_recent = true)
+    Suspicious: Suspicious = true and not malicious
+    """
+
+    emails = argToList(args.get('email'))
+    if len(emails) == 0 or emails is None:
+        raise ValueError('Email(s) not specified')
+
+    email_data_list: List[Dict[str, Any]] = []
+    email_score_list: List[Common.Email] = []
+    for email in emails:
+        email_data = client.get_email_address_reputation(email)
+        email_data_list.append(email_data)
+
+        score = Common.DBotScore.NONE
+        description = f'{INTEGRATION_NAME} returned'
+        suspicious = email_data.get('suspicious')
+        malicious_activity_recent = email_data.get('details.malicious_activity_recent')
+        credentials_leaked_recent = email_data.get('details.credentials_leaked_recent')
+        if not suspicious:
+            score = Common.DBotScore.GOOD
+            description = ''
+        elif malicious_activity_recent or credentials_leaked_recent:
+            if malicious_activity_recent:
+                description += ' malicious_activity_recent'
+            if credentials_leaked_recent:
+                description += ' credentials_leaked_recent'
+            score = Common.DBotScore.BAD
+        else:
+            score = Common.DBotScore.SUSPICIOUS
+            description = ''
+
+        dbot_score = Common.DBotScore(
+            indicator=email,
+            indicator_type=DBotScoreType.EMAIL_ADDRESS,
+            integration_name=INTEGRATION_NAME,
+            score=score,
+            malicious_description=description
+        )
+
+        email_context = Common.Email(
+            email_address=email,
+            dbot_score=dbot_score
+        )
+
+        email_score_list.append(email_context)
+
+    readable_output = tableToMarkdown('Email List', email_data_list)
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix=f'{INTEGRATION_NAME}.EmailScore',
+        outputs_key_field='email',
+        outputs=email_data_list,
+        indicators=email_score_list
```

Comment:
Please split this to multiple `CommandResults` (1 per email) with a single indicator/Human Readable/output per result.
The handling of multiple indicators is due to change as detailed in https://github.com/demisto/etc/issues/29016.
Do note, `return_results` currently does not support list of `CommandResults` - so for now you'll need to handle it via the integration, like the changes that are introduced here: https://github.com/demisto/content/blob/84f6226db4034d85dbfecc60e545d2254aea6ea9/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L2771

Reply:
No problem, Arseny wanted this done for one email in each command anyway.